internal/controller/servicemgr/types.go

```diff
@@ -41,6 +41,7 @@ const (
 	// ConstKeyUserArgs is key of user argument
 	ConstKeyUserArgs = "UserArgs"
 
+	//ConstKeyRequester is key for requester
```

Comment:
```suggestion
	// ConstKeyRequester is key for requester
```

@nitu-s-gupta Same trivial suggestion from me. 😄

Reply:
Done